be/src/olap/schema_change.cpp

```diff
@@ -1661,6 +1677,11 @@ OLAPStatus SchemaChangeHandler::_convert_historical_rowsets(const SchemaChangePa
         goto PROCESS_ALTER_EXIT;
     }
 
+    if (sc_params.new_tablet->tablet_meta()->preferred_rowset_type() == BETA_ROWSET) {
+        // if the tablet meta has preferred_rowset_type field set to BETA_ROWST, just use directly type
+        sc_directly = true;
```

Comment:
If user add a sc_sorting rollup, this is a error.

Reply:
I remove this logic to parse_request, and it will avoid the problem you mentioned.